src/schema/entities.yaml

```diff
@@ -102,13 +102,24 @@ echo:
   name: Echo
   entity: echo
   description: |
-    Multi-echo data MUST be split into one file per echo.
-    Each file shares the same name with the exception of the `_echo-<index>`
-    key/value.
+    If constituents of an entity-linked file collection differ as a
+    function of `EchoTime` acquisition parameter, the `_echo-<index>` key/value
+    pair MUST be used to distinguish individual files.
```

Comment:
I think this revised definition is worded a little confusingly. Rather than "constituents of an entity-linked file collection", I would say "files that are part of an entity-linked file collection". Instead of "differ as a function of `EchoTime`", I think you could say "have different `EchoTime`s.

Reply:
Also, I think the definition should go before the usage and warning. Something like "The `_echo-<index>` key/value pair represents the `EchoTime` metadata field." Then you can go into when and how it should be used.

Reply:
I agree, re-wording this now.